R/build-search-docs.R

```diff
@@ -3,16 +3,19 @@ build_docsearch_json <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
 
   data <- list(
```

Comment:
I just realised I have a local version that I forgot to push — I took the opposite approach of wrapping things that needed to be boxed in lists. That might be a bit cleaner here.

Reply:
done